src/components/PostDetails/PostDetails.scss

```diff
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
```

Comment:
Isn't the name misguiding ? visible list should have display of none ?
I guess you can use just ```PostDetails__list``` for both lists and just add modifier ```--invisible``` to the one that should be hidden.